package.json

```diff
@@ -28,7 +28,7 @@
     "vue-router": "^4.3.0",
     "webmidi": "^3.1.8",
     "xen-dev-utils": "^0.2.9",
-    "xen-midi": "^0.1.3"
+    "xen-midi": "github:xenharmonic-devs/xen-midi"
```

Comment:
Is `xen-midi` good enough now? I should publish current `main` then.

Reply:
It is fine for multichannel use, please publish! I was thinking of adding another export MidiOutMTS which would give an alternative to multichannel PB using Midi Tuning Standard and sysex (opt in popup for browser users): The logic of retuning in Midi 2.0 is almost identical so the code will be re-useable when sw adds Midi 2.0 compatibility (roadmap 4.0???) Concept is to use the note numbers as slots in which the "actual" neareast MIDInote and needed PB are stored. Then one can simply cycle through the MIDI note slots and recover info from them for per-note stuff (NoteOff Velocity, Poly AT etc.)

Reply:
Published. Please update to `"xen-midi": "^0.2.0"`

Please raise a dedicated issue in `xen-midi` about this MIDI 2.0 stuff, so your ideas won't get lost once this PR is closed.